src/dredd-command.coffee

```diff
@@ -9,13 +9,15 @@ spawnArgs = require 'spawn-args'
 Dredd = require './dredd'
 interactiveConfig = require './interactive-config'
 configUtils = require './config-utils'
-logger = require './logger'
+logger = require('./logger')
 
 packageData = require('../package.json')
 
+
```

Comment:
I usually use two lines to separate top-level module content and one line to separate everything else. The CS style guide surprised me by suggesting that after `class` there should be three new lines or something like that, so if that popped up on me, I obeyed, although I don't think 3 lines are in any way better than 2.